### PR TITLE
Pride Flag Overhaul

### PR DESCRIPTION
**Pull request by Sylphet**
## About The Pull Request

Resprites the biodome pride flags, makes them craftable for 3 cloth, and adds a vertical variant so they can be used in more places. You can rotate the flag with alt-click while it's on a wall. Click and drag it to yourself to take it down and fold it up.

## Why It's Good For The Game

The old sprites unfortunately had some serious codersprite going on and clashed really hard with the visual style of the game. Also, new sprites are good and it helps my fellow fruits feel seen on bubberstation thirteen, the furry rp server with antags.

## Proof Of Testing

uncoders your sprite

you know what fuck you

crafting menu

## Changelog

:cl:
add: added vertical variants of the pride flags used in biodome
image: resprited horizontal pride flags used in biodome
/:cl:

### CODE REVIEW

**Comment by ReturnToZender** [comment]
Sexy, can you make these craftable using cloth?

**Comment by Majkl-J** [comment]
> Sexy, can you make these craftable using cloth? and make it so they can be hung on walls like posters?

Out of scope

**Review by Majkl-J — APPROVED** [review]
Love it

**Comment by Sylphet** [comment]
> Sexy, can you make these craftable using cloth? and make it so they can be hung on walls like posters?

Functionality is actually already there for wall hanging, will be doing this tonight ^^

**Comment by Sylphet** [comment]
> Sexy, can you make these craftable using cloth? and make it so they can be hung on walls like posters?

The host moth's will has been done. ♥

**Comment by theselfish** [comment]
Add them to the loadout.

**Comment by Sylphet** [comment]
I'm already adding 7 things to the loadout menu in another PR, these are easy enough to make that I'm not sure it's a good idea to bloat it further.

**Comment by theselfish** [comment]
> I'm already adding 7 things to the loadout menu in another PR, these are easy enough to make that I'm not sure it's a good idea to bloat it further.

All flags but the US flag are in the loadout.
I'd rather see these around than the NCR flag.
It also means that jobs without easy access to cloth can show their support and piss off bigots.

**Comment by Sylphet** [comment]
>  It also means that jobs without easy access to cloth can show their support and piss off bigots.

...Yeah, if we can have roughly 65 plushies of various skyrat statics, we can give marginalized groups more opportunity to express themselves.

The scope creep on this PR though, ughhh

File: modular_zubbers/maps/biodome/pride_flags.dm [hunk 1]
```diff
@@ -3,41 +3,113 @@
...
 	icon = initial(fake_type.icon)
 
+
+// HORIZONTAL
+
 /obj/structure/sign/flag/pride
-	name = "flag of Gay Pride"
-	desc = "The flag of Gay Pride. Hang that rainbow up with pride!"
+	name = "coder pride flag"
+	desc = "You probably shouldn't be seeing this. Yell at the coders about it."
```
Comment (Majkl-J) [inline modular_zubbers/maps/biodome/pride_flags.dm:11]: If you add the purple black checker texture to this one I will love you

Comment (ReturnToZender) [reply modular_zubbers/maps/biodome/pride_flags.dm:11]: Coders need to install Counter Strike: Source

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_pocket.dm [hunk 2]
```diff
@@ -293,6 +293,33 @@ GLOBAL_LIST_INIT(loadout_pocket_items, generate_loadout_items(/datum/loadout_ite
 	name = "Amber Perfume"
 	item_path = /obj/item/perfume/amber
 
+/*
```
Comment (nevimer) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_pocket.dm:296]: Certainly not in modular skyrat

**Review by nevimer — CHANGES_REQUESTED** [review]

**Comment by xXPawnStarrXx** [comment]
Okay. Wild idea, but hear me out...

GAGS Flags?
image

**Pull request merged**